DEVELOPMENT.rst

```diff
@@ -107,32 +107,91 @@ All other commands are the same as for GNU/Linux and Mac OS X.
 a. Running Angular.js test cases
 ''''''''''''''''''''''''''''''''
 
+Run them by starting karma::
+
     $ node_modules/.bin/karma start tests/karma/karma.conf.js
 
 
 Installation OpenSlides in big mode
 ===================================
 
-1. Install PostgreSQL und redis:
+1. Install PostgreSQL and redis
+-------------------------------
+
+    Just apt-get these packets::
+
+    $ sudo apt-get install postgresql redis-server libpq-dev --yes
+
+2. Configure postgresql
```

Comment:
Hmm.. I didn't use postgresql before and googled for 'configure postgresql'. This is (a little bit updated) version of [this result](http://stackoverflow.com/questions/1471571/how-to-configure-postgresql-for-the-first-time).

If you have a lighter quick setup guide please let me know.